brave/brave-core
# Paper button styles

Fixes: https://github.com/brave/brave-browser/issues/3633
Fixes: https://github.com/brave/brave-browser/issues/4852

## Before
<img width="329" alt="Screen Shot 2019-06-11 at 11 28 49 PM" src="https://user-images.githubusercontent.com/29072694/59328222-b4f32380-8ca0-11e9-862b-bc574b61384a.png">
<img width="358" alt="Screen Shot 2019-06-11 at 11 28 31 PM" src="https://user-images.githubusercontent.com/29072694/59328223-b4f32380-8ca0-11e9-9c85-0cd2a00cc7f3.png">


## After
![example2](https://user-images.githubusercontent.com/29072694/59328240-bf152200-8ca0-11e9-8f7f-2e1550150a3c.gif)
![example](https://user-images.githubusercontent.com/29072694/59328241-bf152200-8ca0-11e9-8c1d-44c26b4122b6.gif)


## Test Plan:
1. Go to settings
2. Ensure the buttons in setting match button spec
3. Open the print dialog with File -> Print
4. Ensure the buttons are styled to spec.

The buttons effected by these style updates live in
- Settings panels (not Brave Rewards)
- Print modal window
- Dialogs within settings
